<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameSelection</class>
 <widget class="QDialog" name="gameSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3E424B;;</string>
  </property>
  <widget class="QLabel" name="pannle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #3E424B;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>dripicons-v2</family>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 125, 236, 255);</string>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonTraining">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#buttonTraining{
	background-color: rgba(2, 65, 118, 255);
	color: rgba(255, 255, 255, 200);
	border-radius: 5px;
}

#buttonTraining:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(2, 65, 118, 100);
}

#buttonTraining:hover,
#buttonTraining:focus{
	background-color: rgba(2, 65, 118, 200);
}</string>
   </property>
   <property name="text">
    <string>Training starten</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="title">
    <string>Auf diese Sprache musst du Ãœbersetzen:</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="eng">
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Englisch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ger">
      <property name="styleSheet">
       <string notr="true">color: white;
float: right;</string>
      </property>
      <property name="text">
       <string>Deutsch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="group2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="title">
    <string>Extra Einstellungen</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="time">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>mit Zeitbegrenzung</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="setTime">
      <property name="styleSheet">
       <string notr="true">color: transparent; 
border-bottom-color: transparent; 
border: 0;</string>
      </property>
      <property name="placeholderText">
       <string>Zeit(s) pro Vokabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="errorMsg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red; 
font: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorMsg_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red; 
font: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#close{
	background-color: rgba(0, 0, 0, 0);
	color: rgba(255, 255, 255, 200);
	border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>eng</tabstop>
  <tabstop>ger</tabstop>
  <tabstop>time</tabstop>
  <tabstop>setTime</tabstop>
  <tabstop>buttonTraining</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
